WordPress/gutenberg
# Meta Boxes: Fix WordCounter error when loading meta boxes

Some meta box plugins when loaded in Gutenberg were creating an issue #5118 due to the way we were hacking the `wordcount` WP Core package.

This PR changes the way we generate the `wp.utils` global in the `wp-utils` script by exporting previous scripts available in the global.

**Testing instructions**

 - Install the EditFlow plugin
 - Open the gutenberg editor
 - You shouldn't see any error in the console.
